doc/user/install_guide.md

```diff
@@ -38,6 +38,7 @@ kubectl delete clusterrolebinding etcd-operator
 ```
 
 ## Installation via Helm
+**Disclaimer:** The following Helm chart is an external project not maintained by the etcd-operator maintainers. It may not be up to date and so is not the recommended way to install the etcd-operator.
```

Comment:
i do not think we have any recommended way to install etcd operator. although we have one inside tectonic, but it is not public available. so we should remove the last half sentence?